File: clinichm-ui/src/models/Usuario.ts
export interface Usuario {
  id: number;
  userName: string;
  nombre: string;
  apellido: string;
  password?: string; // Agregar la propiedad opcional password
  rolId: number;
  celular: string;
  sucursalID: number; // Cambiar sucursalId a sucursalID
}

File: clinichm-api/Models/Usuario.cs
using System.ComponentModel.DataAnnotations.Schema;
namespace clinichm_api.Models;

public class Usuario
{
    public int Id {get; set;}

    public string? UserName {get; set;}

    public string? Nombre {get; set;}

    public string? Apellido {get; set;}

    public string? Password {get; set;}

    public int RolId {get; set;}

    public string? Celular {get; set;}

    public int SucursalID {get; set;}
}